src/app.rs

```diff
@@ -176,76 +152,86 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
                 .unwrap_or_default(),
         );
 
-        let app_init_cfg = AppInitCfg {
-            mount_type: MountType::Takeover,
-            phantom: PhantomData,
-            into_after_mount: Box::new({
-                let base_path = Rc::clone(&base_path);
-                move |url: Url,
-                      orders: &mut OrdersContainer<Ms, Mdl, INodes, GMs>|
-                      -> AfterMount<Mdl> {
-                    let url = url.skip_base_path(&base_path);
-                    let model = init(url, orders);
-                    AfterMount::new(model).url_handling(UrlHandling::None)
-                }
-            }) as Box<dyn IntoAfterMount<Ms, Mdl, INodes, GMs>>,
+        let app = Self {
```

Comment:
Removing `App` and `AppCfg` construction abstractions made it easier to make the needed changes. Reverting back to use of the constructors should be trivial.